internal/pkg/services/argus/utils/utils_test.go

```diff
@@ -220,3 +349,87 @@ func TestValidatePlanId(t *testing.T) {
 		})
 	}
 }
+
+func TestMapToUpdateScrapeConfigPayload(t *testing.T) {
+	tests := []struct {
+		description     string
+		resp            *argus.GetScrapeConfigResponse
+		expectedPayload *argus.UpdateScrapeConfigPayload
+		isValid         bool
+	}{
+		{
+			description:     "base case",
+			resp:            fixtureGetScrapeConfigResponse(),
+			expectedPayload: fixtureUpdateScrapeConfigPayload(),
+			isValid:         true,
+		},
+		{
+			description: "nil response",
+			resp:        nil,
+			isValid:     false,
+		},
+		{
+			description: "nil data",
+			resp: &argus.GetScrapeConfigResponse{
+				Data: nil,
+			},
+			isValid: false,
+		},
+		{
+			description: "empty data",
+			resp: &argus.GetScrapeConfigResponse{
+				Data: &argus.Job{},
```

Comment:
This use case is really weird and probably not happenning, but it makes me think that we could maybe check in the generate-payload command if the updateScrapeConfigPayload is empty and return error, because that's not usable at all (it doesn't even have the fields in the resulting json)